spaces/domains.json

```
@@ -228,9 +228,10 @@
   "vote.steakhouse.finance": "vote-steakhouse.eth",
   "vote.flashstake.io": "flashcommunity.eth",
   "vote.fenixtoken.net": "fenixtoken.eth",
+  "vote.supershiba.exchange": "supershiba.eth",
   "vote.liquifi.org": "liquifi.eth",
   "vote.x0swap.com": "x0swap.eth",
   "vote.honeydao.com": "honeydao.eth",
   "gov.kumatoken.com": "kumainu.eth",
-  "gov.beluga.fi": "belugaprotocol.eth"
+  "gov.beluga.fi": "belugaprotocol.eth",
```

Please delete the comma at the end of the last line.
```suggestion
  "gov.beluga.fi": "belugaprotocol.eth"
```